### ISSUE TITLE
Context menu doesn't appear on 0.5.10

### ISSUE THREAD

**Issue by vadiof** [lutris/lutris#4221]
### Bug description

Since 0.5.10 pop-up context menu doesn't appear regardless if I click right mouse button or use keyboard's context menu key. This appears in both Cinnamon and MATE on my machine. On 0.5.9 and earlier versions it worked ok

### How to Reproduce

1. Open Lutris
2. Click by RMB on any game's icon
3. Nothing happens

### Expected behavior

Pop-up context menu should appears

### Log output

```shell
(lutris:473132): Gdk-CRITICAL **: 01:10:09.461: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/lutris/gui/views/base.py", line 52, in popup_contextual_menu
    self.contextual_menu.popup(event, game_actions)
  File "/usr/lib/python3/dist-packages/lutris/gui/widgets/contextual_menu.py", line 53, in popup
    displayed = game_actions.get_displayed_entries()
  File "/usr/lib/python3/dist-packages/lutris/game_actions.py", line 141, in get_displayed_entries
    and not steam_shortcut.all_shortcuts_set(self.game)
  File "/usr/lib/python3/dist-packages/lutris/util/steam/shortcut.py", line 48, in all_shortcuts_set
    shortcut_found = [
  File "/usr/lib/python3/dist-packages/lutris/util/steam/shortcut.py", line 50, in <listcomp>
    if game.name in s['AppName']
KeyError: 'AppName'
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/lutris/gui/lutriswindow.py", line 399, in update_revealer
    self.game_bar = GameBar(game, self.game_actions, self.application)
  File "/usr/lib/python3/dist-packages/lutris/gui/widgets/game_bar.py", line 61, in __init__
    self.update_view()
  File "/usr/lib/python3/dist-packages/lutris/gui/widgets/game_bar.py", line 86, in update_view
    self.play_button = self.get_play_button()
  File "/usr/lib/python3/dist-packages/lutris/gui/widgets/game_bar.py", line 226, in get_play_button
    popover = self.get_popover(self.get_game_buttons(), popover_button)
  File "/usr/lib/python3/dist-packages/lutris/gui/widgets/game_bar.py", line 233, in get_game_buttons
    displayed = self.game_actions.get_displayed_entries()
  File "/usr/lib/python3/dist-packages/lutris/game_actions.py", line 141, in get_displayed_entries
    and not steam_shortcut.all_shortcuts_set(self.game)
  File "/usr/lib/python3/dist-packages/lutris/util/steam/shortcut.py", line 48, in all_shortcuts_set
    shortcut_found = [
  File "/usr/lib/python3/dist-packages/lutris/util/steam/shortcut.py", line 50, in <listcomp>
    if game.name in s['AppName']
KeyError: 'AppName'
```

### System Information

```shell
[System]
OS:              Ubuntu 20.04 focal
Arch:            x86_64
Kernel:          5.13.0-39-generic
Desktop:         X-Cinnamon
Display Server:  x11

[CPU]
Vendor:          AuthenticAMD
Model:           AMD Ryzen 5 2600 Six-Core Processor
Physical cores:  6
Logical cores:   12

[Memory]
RAM:             31.3 GB
Swap:            1.0 GB

[Graphics]
Vendor:          AMD
OpenGL Renderer: Radeon RX 570 Series (POLARIS10, DRM 3.41.0, 5.13.0-39-generic, LLVM 12.0.0)
OpenGL Version:  4.6 (Compatibility Profile) Mesa 21.2.6
OpenGL Core:     4.6 (Core Profile) Mesa 21.2.6
OpenGL ES:       OpenGL ES 3.2 Mesa 21.2.6
Vulkan:          Supported
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by danieljohnson2** [comment]
This looks to me like a dupe of #4195, and a fix for that has already been merged.

Perhaps you could download the 'master' branch and see if it helps? Don't install it, just run the bin/lutris script from the command line. I bet your contexts will be menuing again.

**Comment by strycore** [comment]
Closing as duplicate

**Comment by vadiof** [comment]
@danieljohnson2 

Thanks! Git version fixes the bug for me

**Comment by strycore** [comment]
We'll ship a fix in 0.5.10.1 soon
